fight-service/src/main/resources/application.properties

```diff
@@ -5,3 +5,11 @@ quarkus.http.test-port=0
 
 quarkus.http.cors=true
 quarkus.http.cors.origins=*
+
+
+quarkus.langchain4j.openai.base-url=https://granite-7b-instruct-llm.apps.ai.sandbox2007.opentlc.com/v1
```

Comment:
Should this be in a public repo or a secret?

Reply:
it will be shutdown tonight.